seatunnel-connectors-v2/connector-file/connector-file-ftp/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/sink/ftp/FtpFileSink.java

```diff
@@ -39,11 +45,19 @@ public SinkFileSystemPlugin getSinkFileSystemPlugin() {
     public void prepare(Config pluginConfig) throws PrepareFailException {
 
         super.prepare(pluginConfig);
-        FtpFileUtils.FTP_HOST = String.valueOf(pluginConfig.getString("ftp_host"));
-        FtpFileUtils.FTP_USERNAME = String.valueOf(pluginConfig.getString("ftp_username"));
-        FtpFileUtils.FTP_PASSWORD = String.valueOf(pluginConfig.getString("ftp_password"));
-        FtpFileUtils.FTP_PORT = pluginConfig.getInt("ftp_port");
 
-        FtpFileUtils.getFTPClient();
+        if (pluginConfig.hasPath(FtpConfig.FTP_HOST)) {
```

Comment:
I think these parameters are required for your connector, you should add check logic for these parameters. If user do not  config these parameters you should throw RuntimeExecption.

Reply:
In common module, seatunnel has a util to check config parameters conveniently. This is an example:

```java
        CheckResult result = CheckConfigUtil.checkAllExists(pluginConfig, HttpConfig.URL);
        if (!result.isSuccess()) {
            throw new PrepareFailException(getPluginName(), PluginType.SOURCE, result.getMsg());
        }

```